apps/sasquatch/src/main/java/com/microsoft/appcenter/sasquatch/activities/EventActivity.java

```diff
@@ -73,7 +68,12 @@
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_event);
+
+        @SuppressLint("InflateParams") View topView = getLayoutInflater().inflate(R.layout.activity_log_top, null);
```

Comment:
wasn't that a real warning? what would be the real fix?

Reply:
This is actually an optional parameter. To avoid the `SuppressLint` we can do this like
```
        @SuppressLint("InflateParams") View topView = getLayoutInflater().inflate(R.layout.activity_log_top, null);
        ((LinearLayout) findViewById(R.id.top_layout)).addView(topView);
```
to
```
        LinearLayout topLayout = findViewById(R.id.top_layout);
        @SuppressLint("InflateParams") View topView = getLayoutInflater().inflate(R.layout.activity_log_top, viewGroup);
        topLayout.addView(topView);
```